integration-examples/lambda-vpc-connection-sample/README.md

```diff
@@ -0,0 +1,85 @@
+# AWS Lambda Instrumetation Example with VPC-Connected OpenTelemetry Collector Gateway
+This project is intended as an example implementation of an Otel-instrumented Python Lambda using the [AWS Serverless Application Model (SAM)](https://aws.amazon.com/serverless/sam/).
+The additional AWS Resources included in this template support the Splunk-recommended best practice of sending Lambda telemetry to an OpenTelemetry Collector Gateway.
+
+For additional information about the AWS SAM framework please refer to the [original README file](./ORIGINAL_README.md) generated with this sample's inital call to `sam init`. This is provided for historical purposes and contains some helpful information about AWS SAM.
+
+# Pre-Requisites
+1. Python 3.9 set as the active Python interpreter on your local machine. [Pyenv](https://github.com/pyenv/pyenv) is a great tool for managing different Python versions globalaly, locally for a given directory or a virtual environment.  Installation can be done with [Homebrew](https://github.com/pyenv/pyenv#homebrew-in-macos) on a Mac or with the [installer tool](https://github.com/pyenv/pyenv-installer).  Using whatever Python version management tool you like, ensure that Python 3.9.x is installed.  For example, to use Python 3.9.13 globally on your local machine, execute the following:
+ ```bash
+pyenv install 3.9.13
+pyev global 3.9.13
+```
+Verify the version of your Python interpreter by executing
+```bash
+python3 --version
+```
+If you prefer some other means to manage the active Python interpreter for your shell session, feel free to use what you like :smiley:
+2. Installation and configuration of the [AWS SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
```

Comment:
This numbered list is rendering as a paragraph instead of a list? May need to do the two empty spaces on the line above each numbered item to force the markdown to render as a numbered list?